### ISSUE TITLE
compact is broken?

### ISSUE THREAD

**Issue by azaroth42** [thegetty/crom#37]
```
>>> print model.factory.toString(v)
{"@context":"https://linked.art/ns/v1/linked-art.json","id":"http://lod.example.org/museum/LinguisticObject/1","type":"LinguisticObject","classified_as":[{"id":"http://vocab.getty.edu/aat/300028045","type":"Type","_label":"A/V Content"}]}
>>> print model.factory.toString(v, compact=False)
{
  "id": "http://lod.example.org/museum/LinguisticObject/1", 
  "type": "LinguisticObject"
}
```

**Comment by azaroth42** [comment]
Pretty sure this is fixed in #38

**Comment by azaroth42** [comment]
Fixed :)
